trimmer_decorator.rb

```diff
@@ -0,0 +1,12 @@
+require './base_decorator'
+
+class TrimmerDecorator < BaseDecorator
+  def initialize(nameable)
+    super()
+    @nameable = nameable
+  end
```

Comment:
- [ ] No need to initialize here, as it is inherited by BaseDecorator, so kindly remove it.